src/codegen/fast_math.hpp

```diff
@@ -133,7 +133,7 @@ static inline double vexpm1(double initial_x) {
     double px = std::floor(LOG2E * x + 0.5);
     const int32_t n = px;
 
-    const uint64_t twopnm1 = (((uint64_t)(n - 1)) + 1023) << 52;
+    const uint64_t twopnm1 = (((uint64_t) (n - 1)) + 1023) << 52;
```

Comment:
unrelated / different clang-format?

Reply:
This is because https://github.com/BlueBrain/hpc-coding-conventions/blob/master/cpp/clang-format-7#L73

Reply:
But I revert it